bundles/mapping/mapmodule/mapmodule.olcs.js

```diff
@@ -248,7 +248,7 @@ class MapModuleOlCesium extends MapModuleOl {
 
     getMapZoom () {
         var zoomlevel = this.getMap().getView().getZoom();
-        if (typeof (zoomlevel) === 'undefined') {
+        if (typeof (zoomlevel) === 'undefined' || zoomlevel === 0) {
```

Comment:
Why wouldn't zoomLevel 0 be ok?

Reply:
When camera is "locked" to rotate mode, zoom level is adjusted to zero. I added zoom level 0 check to restore last known zoom level when rotate mode is switched to back default move mode. I will check if i can retain zoom level some other way when user toggles camera modes.